yt_dlp/extractor/listennotes.py

```diff
@@ -25,7 +25,7 @@ class ListenNotesIE(InfoExtractor):
     }]
 
     def _real_extract(self, url):
-        id = self._match_id(url).replace('/', '').rsplit('-', 1)[1]
+        id = self._match_id(url).rsplit('-', 1)[1]
```

Comment:
Why not change the `VALID_URL` to get right id ?. I recommend to use other variable name other than `id`

Reply:
This is done in e7d09bef4197cbb6d20b7e397b98e3cbce57f0fb